components/dashboard/src/App.tsx

```diff
@@ -51,7 +67,7 @@ function App() {
                     <Switch>
                         {user && (
                             <React.Fragment>
-                                <Route path={["/", "/workspaces"]} exact render={
+                                <Route path="/" exact render={
```

Comment:
Did that not work? Then maybe have two entries?

Reply:
It failed the build for me. Will try again and use two entries if it still fails.